howso/client/base.py

```diff
@@ -292,7 +292,8 @@ def create_trainee(
         overwrite_trainee: bool = False,
         persistence: Persistence = "allow",
         project: t.Optional[str | Project] = None,
-        resources: t.Optional[Mapping[str, t.Any]] = None
+        resources: t.Optional[Mapping[str, t.Any]] = None,
+        runtime: t.Optional[TraineeRuntime] = None
```

Comment:
There should be a different schema I think for this parameter, since there are some properties that are not settable, like versions.

Reply:
Should be `TraineeRuntimeOptions`, let me double-check that it's right here